enterprise/jakartaee.platform/build.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<project basedir="." default="build" name="enterprise/jakartaee.platform">
+
+    <import file="../../nbbuild/templates/projectized.xml"/>
+    <property file="nbproject/project.properties" />
+
+    <!-- Hook into harness "-process.release.files" task -->
+    <target name="-process.release.files" depends="prepare-doc"/>
+
+    <!-- Check if generated doc file was already generated -->
+    <target name="-check-prepared-doc">
+        <condition property="jakartaee.platform.doccreated" value="present">
+            <and>
+                <available file="external/generated-jakarta.jakartaee-api-8.0.0-javadoc.jar" />
+            </and>
+        </condition>
+    </target>
+
+    <target name="prepare-doc" depends="-check-prepared-doc" unless="jakartaee.platform.doccreated">
+        <delete file="external/generated-jakarta.jakartaee-api-8.0.0-javadoc.jar" />
+
+        <!-- repackage doc file into the format netbeans used prior to switching to the maven central artefact -->
+        <jar destfile="external/generated-jakarta.jakartaee-api-8.0.0-javadoc.jar">
+            <zipfileset src="external/jakarta.jakartaee-api-8.0.0-javadoc.jar" includes="**" prefix="docs/api" />
+        </jar>
```

Comment:
Please also add a license file for the generated package. See the `j2ee.platform/external` situation as reference. The generated package has the same license as its source, but we need the license information.

Reply:
Did not notice that license file, thanks.